portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase

```diff
@@ -431,6 +431,19 @@ definition {
 		Staging.activateStagingCP(siteName = "Site B Name");
 	}
 
+	@priority = "5"
+	test ImportLARfrom70toCurrent {
```

Comment:
@gaborlovasdotcom One more minor change: Can you fix the capitalization here? See the other test names for example. Feel free to forward the test after this change. Thanks!

Reply:
Done